server/lib/utils.js

```diff
@@ -5,3 +5,24 @@ allowIsBoardAdmin = function(userId, board) {
 allowIsBoardMember = function(userId, board) {
   return board && board.hasMember(userId);
 };
+
+// todo XXX not really server-specific,
+// so move it to a common (client+server) lib?
```

Comment:
By the way, any reason why the methods in this file (allowIsBoardMember and allowIsBoardAdmin) are not defined as Boards.helpers?